PizzaPyWebApp/templates/partials/landing/about_us.html

```diff
@@ -1,49 +1,91 @@
-                class="w-full h-auto xl:w-[90%] xl:max-w-none rounded-lg shadow-lg">
+<div style="margin-bottom: 80px">
+  <div class="flex-col mb-20 md:mb-50">
+    <h2 class="text-4xl font-bold text-yellow-500 text-center">ABOUT US</h2>
+  </div>
+
+  <div class="py-12 px-8 sm:px-12 md:px-16 lg:px-24 xl:px-32">
+    <div
+      class="container mx-auto grid grid-cols-1 lg:grid-cols-2 lg:gap-8 space-x-4"
+    >
+      <!-- Image Section -->
+      <div
+        class="relative overflow-hidden flex items-center justify-center mb-4"
+      >
+        <div class="slideshow-container relative p-2" style="width: 500px">
+          <img
+            src="../../../static/images/events/group.png"
+            alt="Event 3"
+            class="slide rounded-lg transition-opacity duration-700 inset-0 opacity-0"
+          />
+          <img
+            src="../../../static/images/events/Event-2.png"
+            alt="Event 2"
+            class="slide rounded-lg transition-opacity duration-700 absolute inset-0 opacity-0"
+          />
+          <img
+            src="../../../static/images/events/Event-3.png"
+            alt="Event 3"
+            class="slide rounded-lg transition-opacity duration-700 absolute inset-0 opacity-0"
+          />
+          <img
+            src="../../../static/images/events/Event-4.jpg"
+            alt="Event 4"
+            class="slide rounded-lg transition-opacity duration-700 absolute inset-0 opacity-0"
+          />
+          <img
+            src="../../../static/images/events/Event-5.jpg"
+            alt="Event 5"
+            class="slide rounded-lg transition-opacity duration-700 absolute inset-0 opacity-0"
+          />
+          <img
+            src="../../../static/images/events/Event-8.png"
+            alt="Event 8"
+            class="slide rounded-lg transition-opacity duration-700 absolute inset-0 opacity-0"
+          />
         </div>
+      </div>
+      <!-- Text Content Section -->
+      <div>
+        <h1 class="md:mt-4 lg:text-4xl text-xl font-bold">
+          We share solutions and discuss problems.
+        </h1>
+        <p class="mt-8 sm:mt-4 text-md font-sans text-gray-200">
+          Numbered among us are team members from various tech companies,
+          students, freelancers, instructors, and individuals from other
+          industries. As a community, our main focus is on sharing knowledge.
+        </p>
+
+        <a
+          href="https://www.facebook.com/groups/pizzapy.ph/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="inline-block px-6 py-3 mt-8 text-white transition duration-150 ease-in-out bg-indigo-600 border border-transparent rounded-lg hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-700 focus:ring-opacity-50"
+        >
+          Get To Know Us
+        </a>
+      </div>
     </div>
+  </div>
 </div>
-
 <script>
-    document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Consider integrating this approach, as it ensures flexibility by using JSON to store data, simplifying formatting and enabling seamless updates, such as adding new information.

https://github.com/pizzapy/pizzapy-website/blob/main/PizzaPyWebApp/static/data/lang.json

Reply:
@ChristopherCastro69 💯 👍🏻

Reply:
Minor comment on the padding on the left and right portion of the about us component, increase it by a bit

Overall looks good to me

Reply:
@ChristopherCastro69

Reply:
> Consider integrating this approach, as it ensures flexibility by using JSON to store data, simplifying formatting and enabling seamless updates, such as adding new information.
> 
> https://github.com/pizzapy/pizzapy-website/blob/main/PizzaPyWebApp/static/data/lang.json

hala oo sakto sd
gegege

Reply:
![image](https://github.com/user-attachments/assets/37253c97-f517-49bd-bb41-f598ef825d1c)
Resolved conflicts boss @jackjk521 @cristoph143